src/components/ButtonV2.vue

```diff
@@ -3,7 +3,7 @@
     :is="tag"
     :class="rootClasses"
     :to="to || null"
-    :target="$attrs.href ? '_blank' : null"
+    :target="$attrs.href || $attrs.target ? '_blank' : null"
```

Comment:
Allow `target` to be overrided
```suggestion
    :target="$attrs.href && !$attrs.target ? '_blank' : $attrs.target"
```